jenkins/databricks/test.sh

```diff
@@ -19,11 +19,15 @@ set -ex
 
 LOCAL_JAR_PATH=${LOCAL_JAR_PATH:-''}
 SPARK_CONF=${SPARK_CONF:-''}
-BASE_SPARK_VER=${BASE_SPARK_VER:-'3.1.1'}
+BASE_SPARK_VER=${BASE_SPARK_VER:-'3.1.2'}
 [[ -z $SPARK_SHIM_VER ]] && export SPARK_SHIM_VER=spark${BASE_SPARK_VER//.}db
 
+# Try to use "cudf-udf" conda environment for the python cudf-udf tests.
+if [ -d "/databricks/conda/envs/cudf-udf" ]; then
```

Comment:
Uuse "cudf-udf" conda environment (created by jenkins/databricks/init_cudf_udf.sh) for the python cudf-udf tests.